docs/guide/blog.md

```diff
@@ -127,15 +173,19 @@ message Post {
 }
 ```
 
-The contents of `post.proto` are fairly standard. We define a package name (that is used to identify messages, among other things), specify in which Go package new files should be generated, and finally define `message Post`. Now, after we build and start our chain with Starport, the `Post` type will be available.
+The contents of the `post.proto` file are fairly standard. The file defines a package name that is used to identify messages, among other things, specifies the Go package where new files are generated, and finally defines `message Post`. 
 
-### Write Data to the Store
+Each file save triggers an automatic rebuild.  Now, after you build and start your chain with Starport, the `Post` type is available.
```

Comment:
I added this detail, since the rebuild is automatic

Reply:
great 👍